### PR TITLE
Add IO converter to composer json

### PR DESCRIPTION
**Pull request by sebastijansakac**

### CODE REVIEW

File: Extend/Controller/OrderController.php [hunk 1]
```diff
@@ -369,13 +371,52 @@ private function _getCreditCardFormRequestData()
...
+     *
+     * @since 1.3.0
+     */
+    private function _getSupportedLanguageCode()
+    {
+        $wConverter = new WppVTwoConverter();
+        try {
+            $wConverter->init();
+            $wLanguage = $wConverter->convert(
+                $this->_removeSuffix(Registry::getLang()->getLanguageAbbr())
```
Comment (edra-bumci) [inline Extend/Controller/OrderController.php:394]: @sebastijansakac maybe this is too complex and would be better to do smth like:
$langAbbr = $this->_removeSuffix(Registry::getLang()->getLanguageAbbr());
and then
$wLanguage = $wConverter->convert($langAbbr);
but its just a suggestion.

File: Extend/Controller/OrderController.php [hunk 2]
```diff
@@ -369,13 +371,52 @@ private function _getCreditCardFormRequestData()
...
+
+    /**
+     * @param string $oLangCode
+     * @param string $wCutOffPoint
+     *
+     * @return string
+     *
+     * @since 1.3.0
+     */
+    private function _removeSuffix($oLangCode, $wCutOffPoint = '_')
```
Comment (edra-bumci) [inline Extend/Controller/OrderController.php:411]: @sebastijansakac just a question. why are you adding the cutOffPoint as a second param to the function if its not called with this param and then you can just use the '_' directly inside?

Comment (sebastijansakac) [reply Extend/Controller/OrderController.php:411]: It was copy/paste from PS, as mentioned in the ticket. I just adjusted for oxid

File: Extend/Controller/OrderController.php (hunk 1 above)
Comment (sebastijansakac) [reply Extend/Controller/OrderController.php]: Updated as suggested, and to keep consistence with PS

**Comment by tomazpu** [comment]
The branch is red how is it ready to merge?

**Pull request merged**